src/main/kotlin/com/nfeld/jsonpathkt/Token.kt

```diff
@@ -313,7 +308,7 @@ internal data class DeepScanArrayAccessorToken(val indices: List<Int>) : Token {
         }
     }
 
-    override fun read(json: JsonNode): JsonNode? {
```

Comment:
Nice! Not sure how I missed this when I wrote it

Reply:
I think they might have improved something in Kotlin because I wasn't seeing warnings before updating it